python/taichi/main.py

```diff
@@ -26,6 +26,14 @@ def test_python(test_files=None, verbose=False):
     args = [test_dir]
   if verbose:
     args += ['-s']
+  if test_files is None:
+    try:
+      from multiprocessing import cpu_count
+      cpu_count = cpu_count()
+      print('Starting 4 thread(s) to test...')
```

Comment:
`print(f'Starting {cpu_count} testing thread(s)...')`? We should also consider moving this statement out of the `try catch`.